src/everest_models/forward_models.py

```diff
@@ -18,36 +18,26 @@ def get_forward_models() -> List[str]:
     import ert
     from ert import ForwardModelStepDocumentation, ForwardModelStepPlugin
 
-    def build_forward_model_step_plugin(
-        executable_name: str,
-    ) -> Type[ForwardModelStepPlugin]:
-        forward_model_name = (
-            executable_name[3:]
-            if executable_name.startswith("fm_")
-            else executable_name
-        )
-        class_name = "".join(
-            x.capitalize() for x in forward_model_name.lower().split("_")
-        )
+    def build_forward_model_step_plugin(fm_name: str) -> Type[ForwardModelStepPlugin]:
```

Comment:
Not sure, but maybe at least a unit test that produces a correct class type?

Reply:
Added an issue for testing this functionality https://github.com/equinor/everest-models/issues/86